scripts/Inelastic/IndirectImport.py

```diff
@@ -49,13 +50,26 @@ def _os_env():
     return platform.system() + platform.architecture()[0]
 
 
-def _lib_suffix():
-    if platform.system() == "Windows":
-        suffix = "win"
-    elif platform.system() == "Linux":
-        suffix = "lnx"
+def is_pip_version_of_libs():
+    name = 'Quest'
+    if name in sys.modules:
+        return True
+    elif (spec := importlib.util.find_spec(name)) is not None:
```

Comment:
Assignment expressions are neat but I thought they were only added in Python 3.8? As IDaaS only has 3.6 is that not an issue?